siad/daemon.go

```diff
@@ -202,31 +237,28 @@ func startDaemon(config Config) (err error) {
 		tpool,
 		w,
 	)
-	if err != nil {
-		return err
-	}
 
-	// Bootstrap to the network.
-	if !config.Siad.NoBootstrap && g != nil {
-		// connect to 3 random bootstrap nodes
-		perm, err := crypto.Perm(len(modules.BootstrapPeers))
-		if err != nil {
-			return err
-		}
-		for _, i := range perm[:3] {
-			go g.Connect(modules.BootstrapPeers[i])
-		}
-	}
+	// connect the API to the server
+	srv.mux.Handle("/", a)
```

Comment:
ok, so as of this PR, only one handler is being managed by the mux? Which means that this PR doesn't implement the startup status stuff? (ability to at least expose over the API as much granularity as "gateway loaded... consensus loaded... etc., even if we don't have progress within each module)